test/integration/uninstall/test_uninstall.py

```diff
@@ -306,3 +306,36 @@ def test_uninstall_tarantool_no_switch(tt_cmd, tmp_path):
         os.path.join(testdata_path, "bin", "tarantool_2.10.4"),
         os.path.join(testdata_path, "inc", "include", "tarantool_2.10.4")
     )
+
+
+def test_uninstall_tt_missing_version_character(tt_cmd, tmp_path):
+    configPath = os.path.join(tmp_path, config_name)
+    # Create test config.
+    with open(configPath, 'w') as f:
+        f.write(f'env:\n bin_dir: {tmp_path}\n inc_dir:\n')
+
+    # Copying built executable to bin directory,
+    # renaming it to version and change symlink
+    # to emulate that we have installed it like
+    # tt install tt v1.2.3.
+    # We don't really install it because bug we test
+    # is fixed only for current patch. Due to
+    # binary file is changing while running this
+    # bug won't appear only for current and future versions.
+    shutil.copy(tt_cmd, os.path.join(tmp_path, "tt_v1.2.3"))
+    os.symlink(os.path.join(tmp_path, "tt_v1.2.3"),
+               os.path.join(tmp_path, "tt"))
+
+    # Remove not installed program.
+    uninstall_cmd = [tt_cmd, "--cfg", configPath, "uninstall", "tt", "1.2.3"]
```

Comment:
Please, add a test case with v1.2.3.
```suggestion
    uninstall_cmd = [tt_cmd, "--cfg", configPath, "uninstall", "tt", "1.2.3"]
```

Reply:
Added.